docs_0.0.27/content/deployment/mcollective/index.md

```diff
@@ -37,3 +37,6 @@ mcollective::client: true
 mcollective::server: false
 ```
 
+## Managing Choria Modules Yourself
```

Comment:
maybe above the code snippet under `Every Node` add this in a tip like:

```
{{% notice info %}}
The choria/mcollective_choria module has a number of dependencies, if you use R10k to manage environments please be sure to fetch all dependencies.
{{% notice %}}
```

or something along those lines, then it stands out rather than tucked away at the bottom?